js/data/buffer.js

```diff
@@ -64,45 +70,53 @@ function Buffer(options) {
     this._createPushMethod();
 }
 
-// Start Buffer1 compatability code
-Buffer.prototype = {
-
-    get pos() { return this.length * this.itemSize; },
-    get index() { return this.length; },
+/**
+ * Bind this buffer to a WebGL context.
+ * @private
+ * @param gl The WebGL context
+ */
+Buffer.prototype.bind = function(gl) {
+    var type = gl[this.type];
 
-    get arrayType() { return this.type; },
+    if (!this.buffer) {
+        this.buffer = gl.createBuffer();
+        gl.bindBuffer(type, this.buffer);
+        gl.bufferData(type, this.arrayBuffer.slice(0, this.length * this.itemSize), gl.STATIC_DRAW);
```

Comment:
Correct. We used to use a `DataView` with a constrained length to avoid the copy but IE chokes when you pass that to `bufferData`: a119c1c76be318968d0760e9078fbd8e8a49bd18.